crates/rpc/rpc-types/Cargo.toml

```diff
@@ -21,6 +21,7 @@ alloy-rpc-engine-types = { workspace = true, features = ["jsonrpsee-types"] }
 ethereum_ssz_derive = { version = "0.5", optional = true }
 ethereum_ssz = { version = "0.5", optional = true }
 alloy-genesis.workspace = true
+revm-inspectors.workspace = true
```

Comment:
I don't want the dep here we can convert this in the request impl, because we only really need it there
and eventually we will move the otterscan types to alloy and then can do the conversion natively in revm-inspectors

Reply:
Done.